docs/building/windows-core.md

```diff
@@ -85,7 +85,7 @@ You can run our cross-platform Pester tests with `Start-PSPester`.
 Building in Visual Studio
 -----------------------------
 
-We do not recommend building the Poweshell solution from Visual Studio. This may lead to package version mismatches with errors similar to:
+We do not recommend building the Powershell solution from Visual Studio. This may lead to package version mismatches with errors similar to:
```

Comment:
FWIW, it looks like there are about 109 places where you have `Powershell` instead of `PowerShell`.

I'm ok with doing one here, but I think in general I'd rather do Powershell -> PowerShell as a distinct commit.